tool/common/fido2/fido2.go

```diff
@@ -31,42 +32,59 @@ import (
 	"github.com/go-webauthn/webauthn/protocol/webauthncbor"
 	"github.com/go-webauthn/webauthn/protocol/webauthncose"
 	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
 
 	wancli "github.com/gravitational/teleport/lib/auth/webauthncli"
 )
 
-type fido2Command struct {
-	diag   *fido2DiagCommand
-	attobj *fido2AttobjCommand
+// Command implements the "fido2" hidden/utility commands.
+type Command struct {
+	Diag   *DiagCommand
+	Attobj *AttobjCommand
 }
 
-func newFIDO2Command(app *kingpin.Application) *fido2Command {
-	root := &fido2Command{
-		diag:   &fido2DiagCommand{},
-		attobj: &fido2AttobjCommand{},
+// NewCommand creates a new [Command] instance.
+func NewCommand(app *kingpin.Application) *Command {
+	root := &Command{
+		Diag:   &DiagCommand{},
+		Attobj: &AttobjCommand{},
 	}
 
 	f2 := app.Command("fido2", "FIDO2 commands").Hidden()
 
 	diag := f2.Command("diag", "Run FIDO2 diagnostics").Hidden()
-	root.diag.CmdClause = diag
+	root.Diag.CmdClause = diag
 
 	attObj := f2.Command("attobj", "Parse a stored attestation object").Hidden()
 	attObj.
 		Arg("att-obj", "Attestation object encoded in base64 standard or RawURL").
 		Required().
-		StringVar(&root.attobj.attObjB64)
-	root.attobj.CmdClause = attObj
+		StringVar(&root.Attobj.attObjB64)
+	root.Attobj.CmdClause = attObj
 
 	return root
 }
 
-type fido2DiagCommand struct {
+// TryRun attempts to execute a "fido2" command. Used by tctl.
+func (c *Command) TryRun(ctx context.Context, selectedCommand string) (match bool, err error) {
```

Comment:
I've kept the "Try" methods in here, even though they don't exactly match CLICommand, as it's easier to spot them in this file if/when we add new subcommands.